package.json

```diff
@@ -4,11 +4,13 @@
   "description": "Mozilla's Donation Forms",
   "main": "start.js",
   "scripts": {
-    "start": "npm-run-all --parallel build:l10njson watch:js server",
+    "start": "npm run clean && npm-run-all --parallel build:l10njson watch:js server",
+    "clean": "npm run rebuild && shx rm public/main*.js && shx rm public/style*.css",
     "build:l10njson": "pontoon-to-json --dest ./public",
     "build:exchangerates": "node scripts/exchangerates.js",
     "build:babel": "babel src --compact=true -s --out-dir dist",
     "build:webpack": "webpack --optimize-minimize --optimize-dedupe",
+    "rebuild": "rimraf ./dist && npm-run-all build:*",
```

Comment:
any reason we're using `rimraf` instead of `shx rm -rf`? `shx`/`shelljs` use `rimraf` internally as far as I can tell

Reply:
`shx rm -f` did not want to actually delete a directory on windows for me.

Reply:
what about with the `-r` flag as well?

Reply:
haven't tried, but if shx already uses rimraf, then npm's deduplication should make using rimraf explicitly irrelevant.